src/enkf/params.f90

```
@@ -681,10 +675,6 @@ subroutine read_namelist()
        letkf_flag) then
      print *,'warning: no time localization in LETKF!'
    endif
-   if ((write_ensmean .and. pseudo_rh) .and. .not. use_qsatensmean) then
-      print *,'write_ensmean=T requires use_qsatensmean=T when pseudo_rh=T'
-      call stop2(19)
-   endif
```

Do we need to raise an error and stop if write_ensmean=T and pseudo_RH=T (just remove the .and. not. use_qsatensmean part)?

---

Done